### PR TITLE
Update hip-1.md

### PR DESCRIPTION
**Pull request by bmgentile**
This is the addition of "User Stories" to the successful HIP proposal template.

**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

Signed-off-by: Brady Gentile <brady@hedera.com>

### CODE REVIEW

File: HIP/hip-1.md [hunk 1]
```diff
@@ -151,26 +151,28 @@ Each HIP should have the following parts/sections:
 4. Rationale -- The rationale fleshes out the specification by describing why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages.
 
     The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during the discussion.
+    
+5. User stories -- Provide a list of "user stories" to express how this feature, functionality, improvement, or tool will be used by the end user. Template for user story: “As (user persona), I want (to perform this action) so that (I can accomplish this goal).”
```
Comment (steven-sheehy) [inline HIP/hip-1.md:155]: We should also update the [HIP template](https://github.com/hashgraph/hedera-improvement-proposal/blob/master/hip-0000-template.md).

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

**Comment by steven-sheehy** [comment]
Also need to pass the DCO check by signing off your commits.

File: HIP/hip-1.md (hunk 1 above)
Comment (bmgentile) [reply HIP/hip-1.md:155]: +1 did an update in a PR here: https://github.com/hashgraph/hedera-improvement-proposal/pull/124

**Comment by bmgentile** [comment]
Hey @steven-sheehy how do we get this approved / added? Also this one: https://github.com/hashgraph/hedera-improvement-proposal/pull/124

Am I blocking anything?

**Comment by steven-sheehy** [comment]
See my note about DCO

**Comment by bmgentile** [comment]
-s

File: HIP/hip-1.md (hunk 1 above)
Comment (bmgentile) [reply HIP/hip-1.md:155]: > See my note about DCO

Ah, thanks — having trouble figuring out how to DCO (haven't used that feature before). Where do I add my DCO? Can I do this manually by just appending "Signed-off-by: Brady Gentile brady@hedera.com" to my comment?

Comment (steven-sheehy) [reply HIP/hip-1.md:155]: You can just copy and paste the commands [here](https://github.com/hashgraph/hedera-improvement-proposal/pull/123/checks?check_run_id=3174345013) on the command line.

Comment (steven-sheehy) [reply HIP/hip-1.md:155]: For new commits, just always run `git commit --signoff`.

**Review by sergmetelin — APPROVED** [review]

**Pull request closed without merging**